mono/metadata/mono-hash.c

```diff
@@ -481,10 +481,17 @@ void
 mono_g_hash_table_insert (MonoGHashTable *h, gpointer k, gpointer v)
```

Comment:
A later PR will move this to `external-only.c`, and build a runtime variant without it?

Reply:
@jaykrell my fault. forgot we have that now.
@CoffeeFlux let's move these.

Reply:
everything should now be moved over, and it Builds For Me(tm)